internal/system/agent/init.go

```diff
@@ -146,13 +148,22 @@ func setLoggingTarget() string {
 
 func initializeClients(useConsul bool) {
 	// Create notification client
-	params := types.EndpointParams{
+	paramsNotification := types.EndpointParams{
 		ServiceKey:  internal.SupportNotificationsServiceKey,
 		Path:        "/",
 		UseRegistry: useConsul,
 		Url:         Configuration.Clients["Notifications"].Url(),
-		Interval:    internal.ClientMonitorDefault,
+		Interval:    Configuration.Service.ClientMonitor,
 	}
+	nc = notifications.NewNotificationsClient(paramsNotification, startup.Endpoint{})
 
-	nc = notifications.NewNotificationsClient(params, startup.Endpoint{})
+	// Create command client
+	paramsCommand := types.EndpointParams{
+		ServiceKey:  internal.CoreCommandServiceKey,
+		Path:        "/",
+		UseRegistry: useConsul,
+		Url:         Configuration.Clients["Command"].Url(),
+		Interval:    Configuration.Service.ClientMonitor,
```

Comment:
Use internal.ClientMonitorDefault since there is no [Service] node in the configuration.toml

Reply:
I do see a [Service] node in the configuration.toml each for the Core Command and Support Notifications Microservices. Let's confirm this; I'll synch up with you, thanks.